deob-bytecode/src/main/kotlin/org/openrs2/deob/bytecode/Profile.kt

```diff
@@ -4,10 +4,25 @@ import org.openrs2.asm.filter.GlobClassFilter
 import org.openrs2.asm.filter.GlobMemberFilter
 
 public class Profile(
+    public val libraries: Map<String, LibraryProfile>,
+    public val dependencies: Map<String, DependencyProfile>?,
     public val excludedClasses: GlobClassFilter,
     public val excludedMethods: GlobMemberFilter,
     public val excludedFields: GlobMemberFilter,
     public val entryPoints: GlobMemberFilter,
     public val scrambledLibraries: Set<String>,
     public val maxObfuscatedNameLen: Int
-)
+) {
+    public class LibraryProfile(
+        public val format: String?,
+        public val file: String?,
+        public val requires: Set<String>?,
+        public val signedMove: Set<String>?,
```

Comment:
Given this always has to either have zero or two elements, I think it'd be nicer to have them as individual fields rather than a set.